openapi/__tests__/definitions.test.ts

```diff
@@ -232,6 +235,50 @@ const aPayee: Payee = { fiscal_code: anOrganizationFiscalCode };
 
 const aThirdPartyId = "aThirdPartyId" as NonEmptyString;
 
+describe("BaseEnrichedMessage definition", () => {
+  it("should decode a BaseEnrichedMessage", () => {
+    const aBaseEnrichedMessage = {
+      ...aMessageWithoutContent,
+      service_name: "aService",
+      organization_name: aService.organizationName,
+      message_title: "aTitle",
+    };
+
+    expect(
+      E.isRight(BaseEnrichedMessage.decode(aBaseEnrichedMessage))
+    ).toBe(true);
+  });
+});
+
+describe("EnrichedMessageWithOrganizationFiscalCode definition", () => {
+  it("should fail decoding a EnrichedMessageWithOrganizationFiscalCode if the CF is not provided", () => {
```

Comment:
```suggestion
  it("should fail decoding a EnrichedMessageWithOrganizationFiscalCode if the organization fiscal code is not provided", () => {
```
I'd expand the name because CF is misleading with the one of the citizen.